views/default/elements/layout.css.php

```diff
@@ -86,6 +86,45 @@
 .elgg-layout {
 	min-height: 360px;
 }
+.elgg-layout-one-sidebar,
+.elgg-layout-two-sidebar {
+	display: -webkit-box;
+	display: -webkit-flex;
+	display: -ms-flexbox;
+	display: flex;
+	-webkit-box-align: start;
+	-webkit-align-items: flex-start;
+	-ms-flex-align: start;
+	align-items: flex-start;
+}
+.elgg-layout-one-sidebar > .elgg-body,
+.elgg-layout-two-sidebar > .elgg-body {
+	-webkit-box-flex: 1;
+	-webkit-flex: 1;
+	-ms-flex: 1;
+	flex: 1;
+}
+.elgg-layout-one-sidebar > .elgg-sidebar {
+	float: none;
+	-webkit-box-ordinal-group: 2;
+	-webkit-order: 1;
+	-ms-flex-order: 1;
+	order: 1;
+}
+.elgg-layout-two-sidebar > .elgg-sidebar {
```

Comment:
I think I changed the sidebar position in the layout view. We need to add media queries to reorder them on mobile.